t/ui/15-search.t

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env perl
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use Test::Most;
+
+use Test::Mojo;
+use Test::Warnings ':report_warnings';
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use OpenQA::Test::Case;
+use OpenQA::Client;
+
+use OpenQA::SeleniumTest;
+
+OpenQA::Test::Case->new->init_data(skip_fixtures => 1);
+
+my $driver = call_driver();
+if (!$driver) {
+    plan skip_all => $OpenQA::SeleniumTest::drivermissing;
+    exit(0);
+}
+
+my $t = Test::Mojo->new('OpenQA::WebAPI');
+# we need to talk to the phantom instance or else we're using wrong database
+my $url = 'http://localhost:' . OpenQA::SeleniumTest::get_mojoport;
+
+subtest 'Perl modules' => sub {
+    my $search = $driver->find_element_by_id('global-search');
+    $search->click();
+    is $search->get_text(), '', 'empty search entry by default';
+
+    $search->send_keys('timezone');
+    $search->send_keys(Selenium::Remote::WDKeys->KEYS->{'enter'});
+    wait_for_ajax(msg => 'wait for search results to update');
+
+    like $driver->get_title(), qr/Search/, 'search shown' or return;
+    my $header = $driver->find_element_by_id('results-heading');
+    is $header->get_text(), '2 matches found', 'number of results in header';
+    my $results = $driver->find_element_by_id('results');
+    my @entries = $results->children('.list-group-item');
+    is scalar @entries, 2, 'two elements' or return;
+
+    my $first = $entries[0];
+    is $first->child('#occurrence')->get_text(), 'opensuse/tests/installation/installer_timezone.pm',
```

Comment:
This now needs to be `.occurrence`. The same counts for the other occurrences.

Reply:
D'uh... I wonder how my tests did not fail locally, though... will fix in any case

Reply:
Not sure I've noticed page hangs but I'll play with it some more and see if I notice it

Reply:
I've noticed them when testing locally (see my last review comment). It seems that the non-jQuery version is a little bit more efficient but it still hangs when searching for "test".

Reply:
Yeah. I see now what you mean, with more generic terms. Also ran into a case where the data was so much that the network cut off part of the JSON. I'll propose something as a follow-up (and this will become even more noticible with other types of matches anyway so thank you for raising it early on).